### PR TITLE
doc: calculated ROI for the business model

### PR DESCRIPTION
**Pull request by alejandromd**
# Description
calculated ROI for the business model

# Checklist:
- [x] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [x] I have checked that there is no fragile information published unintentionally.

# Screenshot:
image

### CODE REVIEW

File: docs/03. Planificación/Modelo de negocio.mdx [hunk 1]
```diff
@@ -3,21 +3,31 @@
...
+El plan de negocio se centrará en desarrollar una PWA eficiente y fácil de usar que facilite la gestión de las operaciones de la ONG y las donaciones. Los ingresos para cubrir los costos de desarrollo y mantenimiento de la PWA provendrán de un crecimiento lineal del número de ONGs clientes, lo que significa que los ingresos aumentarán de manera constante con el tiempo. En el primer año, se espera vender la aplicación a 5 ONGs, generando ingresos de 13.750 €. En los años siguientes, se espera que este número aumente de forma lineal, llegando a 20 ONGs en el cuarto año y generando un total de 143.500 € en ingresos durante este período. Para maximizar la eficiencia de costos, se ha llegado a la conclusión de que hay minimizar los costos de transacción con los procesadores de pagos, por lo que haremos uso de una pasarela de pago con Stripe.
 
 ### 1.1 Aspectos clave del modelo de negocio
 
-- Fuente de ingresos: Un 5% de cada donación realizada.
-
 - Propuesta de valor: Proporcionar una plataforma para facilitar la gestión de las operaciones de la ONG y las donaciones.
 
 - Actividades clave: Desarrollo y mantenimiento de la PWA, gestión de las donaciones, soporte al cliente.
```
Comment (alvarobernal2412) [inline docs/03. Planificación/Modelo de negocio.mdx:16]: We do not manage donations

File: docs/03. Planificación/Modelo de negocio.mdx [hunk 2]
```diff
@@ -3,21 +3,31 @@
...
 
 - Socios clave: Procesadores de pagos para gestionar las donaciones (Stripe).
```
Comment (alvarobernal2412) [inline docs/03. Planificación/Modelo de negocio.mdx:18]: It is interesting to note that in this case winning non-profit investors can benefit us to better help these NGOs, which is an important cause due to the contribution of NGOs to society.

**Review by alvarobernal2412 — CHANGES_REQUESTED** [review]
The change you have made is correct, although some improvements should be made.

File: docs/03. Planificación/Modelo de negocio.mdx (hunk 1 above)
Comment (alejandromd) [reply docs/03. Planificación/Modelo de negocio.mdx:16]: I don't understand, we don't manage donations but I have deleted that line, what do you mean?

File: docs/03. Planificación/Modelo de negocio.mdx (hunk 2 above)
Comment (alejandromd) [reply docs/03. Planificación/Modelo de negocio.mdx:20]: I have taken your comment into account, you can check again and let me know if you see any further improvements.

File: docs/03. Planificación/Modelo de negocio.mdx (hunk 1 above)
Comment (alvarobernal2412) [reply docs/03. Planificación/Modelo de negocio.mdx:18]: Captura de pantalla 2024-05-01 a las 18 37 04

Comment (alejandromd) [reply docs/03. Planificación/Modelo de negocio.mdx]: thank you, it's fixed now

**Pull request merged**